Dockerfile

```diff
@@ -28,22 +28,21 @@ RUN npm ci --omit=dev --ignore-scripts \
   && npm run build -- --base=$UI_PUBLIC_URL
 
 ### Build API ###
-FROM --platform=${BUILDPLATFORM:-linux/amd64} debian:bookworm AS api
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.22-bookworm AS api
 ARG TARGETPLATFORM
 
-COPY docker/install_build_dependencies.sh /tmp/
-COPY docker/go_wrapper.sh /go/bin/go
+COPY scripts /tmp/scripts
 COPY api /app
 WORKDIR /app
 
 ENV GOPATH="/go"
 ENV PATH="${GOPATH}/bin:${PATH}"
-ENV CGO_ENABLED 1
+ENV CGO_ENABLED=1
 
 # Download dependencies
-RUN chmod +x /tmp/install_build_dependencies.sh \
-  && chmod +x /go/bin/go \
-  && /tmp/install_build_dependencies.sh \
+RUN chmod +x /tmp/scripts/*.sh \
+  && /tmp/scripts/install_build_dependencies.sh \
+  && . /tmp/scripts/set_go_env.sh \
```

Comment:
I'd prefer the more explicit form: `source /tmp/scripts/set_go_env.sh`: both work the same (as far as I know), but this is better readable and easier to understand)

Reply:
I tried `source` before. `source` doesn't work with `/bin/sh`. It's a command in `bash`. As `docker build` uses `/bin/sh` by default, it can't use it here.

Reply:
oh, good to know)
But isn't this should instruct the docker build to use bash:
`SHELL ["/bin/bash", "-o", "pipefail", "-c"]`

I've added it on the line 63, as a fix for a linter warning, but maybe it makes sense to add the same line before each RUN with more than 1 command inside?

Reply:
I didn't check the detail. When I tried with `source`, the error showed it was using `/bin/sh`. I don't keep that log. Let me try it again.